pkg/sql/sqlbase/type_desc.go

```diff
@@ -253,26 +253,9 @@ func (desc *MutableTypeDescriptor) IsNew() bool {
 	return desc.ClusterVersion == nil
 }
 
-// AddEnumValue adds an enum member to the type.
+// AddEnumValue assumes that the type is an enum, and that the new value
+// doesn't exist already in the enum.
```

Comment:
Last nit -- don't also remove the comment that actually tells what the function does!

Reply:
fixed